src/queries.js

```diff
@@ -20,7 +20,7 @@ function queryAllLabelsByText(
   const matcher = exact ? matches : fuzzyMatches
   const matchNormalizer = makeNormalizer({collapseWhitespace, trim, normalizer})
   return Array.from(container.querySelectorAll('label')).filter(label =>
-    matcher(label.textContent, label, text, matchNormalizer),
+    matcher(getNodeText(label), label, text, matchNormalizer),
```

Comment:
Don't forget that you'll need to update the `getNodeText` function to use `innerText` :+1: